sims/vcs/Makefile

```diff
@@ -36,22 +39,97 @@ debug: $(sim_debug)
 #########################################################################################
 include $(base_dir)/common.mk
 
+#########################################################################################
+# verilator-specific user-interface variables and commands
+#########################################################################################
+HELP_COMPILATION_VARIABLES +=
```

Comment:
Should this be duplicated across VCS/Verilator?

Reply:
I moved it to `variables.mk` to reduce duplication.